src/app/components/settings/settings.component.ts

```diff
@@ -0,0 +1,30 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-settings',
+  templateUrl: './settings.component.html',
+  styleUrls: ['./settings.component.scss']
+})
+export class SettingsComponent implements OnInit {
+  logoPath:string = "resources/pics/logo.svg";
+
+  userData = {
+    name:String,
+    surname:String,
+    api:String
+  }
+
+  jsonUserData:any
+
+  constructor() { }
+
+  ngOnInit(): void {
+    this.jsonUserData = localStorage.getItem('UserData')
+    this.userData = JSON.parse(this.jsonUserData);
+  }
+
+  onSubmit(){
+    localStorage.setItem('UserData', JSON.stringify(this.userData));
```

Comment:
looks like it doesn't work, I get error on submit

![image](https://user-images.githubusercontent.com/17142002/151666211-b3f6b479-253f-437d-af8d-42df8ceff35b.png)